filters/general/filter_55_1Hosts_Pro/configuration.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "1Hosts (Pro)",
+    "description": "List for blocking pesky ads, trackers, and malware. Strict version: has some minimal breakages, prioritizes privacy & safety (adblocking) over UX e.g blocks graph.facebook.com.",
+    "homepage": "https://badmojr.github.io/1Hosts/",
+    "sources": [
+      {
+        "name": "1Hosts (Pro)",
+        "source": "https://o0.pages.dev/Pro/adblock.txt",
+        "type": "adblock",
+        "transformations": [
+          "RemoveModifiers",
```

Comment:
There are no rules with modifiers in the source file, please remove this transformation.

Reply:
I have updated the file with your suggestions. Thanks.